website/docs/r/api_gateway_rest_api.html.markdown

```diff
@@ -12,13 +12,30 @@ Provides an API Gateway REST API.
 
 ## Example Usage
 
+### Defining REST API
+
 ```hcl
 resource "aws_api_gateway_rest_api" "MyDemoAPI" {
   name        = "MyDemoAPI"
   description = "This is my API for demonstration purposes"
 }
 ```
 
+### Using REST API execution_arn to specify Lambda permissions
```

Comment:
Can you please move this example to the `aws_lambda_permission` resource documentation instead?